src/components/Search/Search.js

```diff
@@ -20,9 +19,9 @@ const Search = (props) => {
 const styles = StyleSheet.create({
   inputContainer: {
     height: 48,
-    backgroundColor: COLORS.white,
+    backgroundColor: '#FFFFFF',
     borderWidth: 1,
-    borderColor: COLORS.silver,
+    borderColor: '#BDBDBD',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color values could be replaced with a centralized color configuration. This would make it easier to maintain and update the color scheme in the future.
```diff
-    backgroundColor: '#FFFFFF',
-    borderWidth: 1,
-    borderColor: '#BDBDBD',
+    backgroundColor: colors.white,
+    borderWidth: 1,
+    borderColor: colors.silver,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->